src/Services/LoggingService.php

```diff
@@ -116,17 +117,26 @@ private function getPlentyMarketLoggerInstance(string $method, string $reference
   }
 
   /**
-   * Maps data from the he associative array that Wayfair provides to the PlentyMarkets logging API's inputs
+   * Maps data from the associative array that Wayfair provides to the PlentyMarkets logging API's inputs
    *
    * @param $loggingInfo
    *
    * @return array
    */
   public function extractVars($loggingInfo): array {
+    $truncatedMsg =[];
+
+
     $additionalInfo = $loggingInfo['additionalInfo'] ?? [];
     $method = $loggingInfo['method'] ?? null;
     $referenceType = $loggingInfo['referenceType'] ?? null;
     $referenceValue = (int) $loggingInfo['referenceValue'] ?? null;
+
+    if (strlen(json_encode($loggingInfo['additionalInfo'])) > self::STRING_LIMIT) {
```

Comment:
the entirety of the log entry must fit within the limit, not just the `additionalInfo` array.

Reply:
I understand that, but I initially used the the additionalInfo property as my check because that was where the bulk of the data is located

Reply:
if you're going to only use this one part then you may need to adjust the limit to match?

Reply:
I made the change you suggested, just wanted you to know why I used the above